hack/run.sh

```diff
@@ -96,6 +95,8 @@ elif [[ "${action}" == "benchmark-filter" ]]; then
   "${ROOT_DIR}/data-plane/benchmarks/run.sh" "$2"
 elif [[ "${action}" == "benchmark-filters" ]]; then
   "${ROOT_DIR}/data-plane/benchmarks/run.sh"
+elif [[ "${action}" == "./hack/update-codegen.sh" || "${action}" == "./hack/run.sh format-java" ]]; then
+    "${ROOT_DIR}"/mvnw spotless:apply
```

Comment:
I don't see this file

Reply:
Yeah I think I made a mistake when I commented on the initial issue - this is my bad!

Reply:
Can you tell me how should I correct it?